src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -579,4 +555,8 @@ private boolean isAnyNodeHandshaking(Jedis node) {
     return false;
   }
 
+  private HostAndPort generateHostAndPort(List<Object> hostInfos) {
```

Comment:
@marcosnils JedisClusterInfoCache already has generateHostAndPort() as private method. It's private method. ;)